vme/src/structs.cpp

```diff
@@ -495,7 +495,7 @@ unit_fptr::unit_fptr()
 
 unit_fptr::~unit_fptr()
 {
-    data = nullptr;
+    free(data);
```

Comment:
Ho hum... ? Sure this is right? :-)) Looks like it could have adverse side-effects. Did we really forget to free the data in the past?